mission_control/test/mission_interface.py

```diff
@@ -128,4 +128,5 @@ def read_behavior_parameters(self, mission_behavior):
 
     def get_behavior_parameter(self, behavior_parameter):
         # return the parameter of the mission (mission/behavior/parameter)
-        return float(self.mission_behavior_parameters.attrib.get(behavior_parameter))
+        parameter = self.mission_behavior_parameters.attrib.get(behavior_parameter)
+        return float(parameter) if parameter else None
```

Comment:
@patapufate 0 will be coerced to False here. Also, assuming the parameter value will always be a number is a bit limiting. Consider:

```python
def get_behavior_parameter(self, name):
    return self.mission_behavior_parameters.attrib.get(name)
```

and deferring type coercion to the caller.